flux-kustomize-controller.yaml

```diff
@@ -38,7 +38,7 @@ pipeline:
 
   - uses: go/build
     with:
-      ldflags: -s -w -X main.Version=${{package.version}}
+      ldflags: -s -w
```

Comment:
I think this change crept in accidentally?

Reply:
🤦‍♂️ whoops